lib/docker-sync/watch_strategy/unison.rb

```diff
@@ -0,0 +1,45 @@
+require 'thor/shell'
+require 'docker-sync/execution'
+require 'docker-sync/preconditions'
+require 'docker-sync/sync_strategy/unison-unox'
+
+module Docker_Sync
+  module WatchStrategy
+    class Unison
+      include Execution
+
+      @options
+      @sync_name
+      @watch_fork
+
+      def initialize(sync_name, options)
+        @options = options
+        @sync_name = sync_name
+        @watch_fork = nil
+        @unison = Docker_Sync::SyncStrategy::Unison_Unox.new(@sync_name, @options)
```

Comment:
This doesn't start the `sync` process, this only instantiate the same class.

Basically running the watcher is the same command as the sync but with the `-repeat watch` option. To ensure that all options are well used, reuse a maximum of code and stick to the strategy pattern, I thought it was the best way to do it.